stakingModel/test/UBIScheme.test.js

```diff
@@ -166,23 +174,22 @@ contract(
     await identity.addWhitelisted(claimer4);
     let claimer3BalanceBefore = await goodDollar.balanceOf(claimer3);
     let claimer4BalanceBefore = await goodDollar.balanceOf(claimer4);
-    await ubi.distribute([claimer3, claimer4]);
+    await ubi.distribute(0,10);
     let claimer3BalanceAfter = await goodDollar.balanceOf(claimer3);
     let claimer4BalanceAfter = await goodDollar.balanceOf(claimer4);
     expect(claimer3BalanceAfter.toNumber()).to.be.equal(claimer3BalanceBefore.toNumber());
     expect(claimer4BalanceAfter.toNumber()).to.be.equal(claimer4BalanceBefore.toNumber());
   });
 
-  it("should not be able to execute claim after the user received tokens since distribute had been executed", async () => {
+  it("should not be able to execute claim after the user received tokens because of distribute had been executed", async () => {
```

Comment:
i think you need to rethink this test case what does it test and how and give better name

Reply:
updated 3d0923a2e6e7579d4047bb53596b808294f64ef0